drivers/sensor/bmm150/bmm150.c

```diff
@@ -0,0 +1,602 @@
+/* bmm150.c - Driver for Bosch BMM150 Geomagnetic Sensor */
+
+/*
+ * Copyright (c) 2017 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "bmm150.h"
+
+static const struct {
+	int freq;
+	u8_t reg_val;
+} bmm150_samp_freq_table[] = { {2, 0x01},
```

Comment:
bad alignment:
`} bmm150_samp_freq_table[] = {
        { 2, 0x01 },
        { 6, 0x02 },
...
};
`

Of course with one tab instead of the 8 space I had to use here.